### PR TITLE
Task content and images

### PR DESCRIPTION
**Pull request by JamesCullen2**
Kyam, mainly focused on adding about me text & changing icons on top nav bar, thanks, J

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,13 +11,19 @@
...
                     <li class="nav-buttons">
-                        <a href="#portfolio">portfolio</a>
+                        <a href="#portfolio">
+                        <img src="nav-bar-icon-portfolio.png" alt="James Cullen Portfolio">
+                        </a>
                     </li>
                     <li class="nav-buttons">
-                        <a href="#about">about</a>
+                        <a href="#about">
+                        <img src="nav-bar-icon-about-me-link.png" alt="James Cullen About Me">
```
Comment (kyam-mayden) [inline index.html]: IMG tag needs to be indented, also on line 15 and 20

File: styles.css [hunk 2]
```diff
@@ -42,9 +42,9 @@ li {
 
 .nav-buttons a:hover {
     color: black;
-    background-color: white;
-    border-style: solid;
-    border-color: white;
+    /*background-color: black;*/
+    /*border-style: solid;*/
+    /*border-color: black;*/
```
Comment (kyam-mayden) [inline styles.css]: hex colours needed

File: styles.css [hunk 3]
```diff
@@ -42,9 +42,9 @@ li {
 
 .nav-buttons a:hover {
     color: black;
-    background-color: white;
-    border-style: solid;
-    border-color: white;
+    /*background-color: black;*/
```
Comment (kyam-mayden) [inline styles.css]: Hex

File: styles.css [hunk 4]
```diff
@@ -59,19 +59,31 @@ section {
     max-width: 150px;
 }
 
+.footer-nav-bar {
+    float: left;
```
Comment (kyam-mayden) [inline styles.css:60]: Possible redundant code for same solution

File: styles.css [hunk 5]
```diff
@@ -96,10 +117,25 @@ section {
...
+        overflow: hidden;
+        position: relative;
+        display: inline-block;
+        width: 100%;
+        height: 30vh;
     }
-}
+    .footer-nav-bar {
+        float: left;
+        margin: 0 auto;
```
Comment (kyam-mayden) [inline styles.css]: possible redundancy

**Review by kyam-mayden — CHANGES_REQUESTED** [review]
Only a few recommendations 👍

File: index.html (hunk 1 above)
Comment (JamesCullen2) [reply index.html]: done

File: styles.css (hunk 2 above)
Comment (JamesCullen2) [reply styles.css]: done

File: styles.css (hunk 3 above)
Comment (JamesCullen2) [reply styles.css]: done

File: styles.css (hunk 4 above)
Comment (JamesCullen2) [reply styles.css:60]: ommitted float:left

File: styles.css (hunk 5 above)
Comment (JamesCullen2) [reply styles.css]: ommitted

File: styles.css [hunk 6]
```diff
@@ -34,14 +34,14 @@ li {
...
+    color: #FFFFFF;
     display: inline-block;
     padding: 15px;
     text-decoration: none;
 }
 
 .nav-buttons a:hover {
-    color: black;
+    color: #000000;
     /*background-color: black;*/
```
Comment (kyam-mayden) [inline styles.css]: Commented out code?

Comment (JamesCullen2) [reply styles.css]: done - thanks, Kyam.

**Review by kyam-mayden — APPROVED** [review]
All good 👍

**Pull request merged**